Overview.src.html

```diff
@@ -3391,21 +3456,23 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <dl class=switch>
    <dt><code data-anolis-spec=fileapi>Blob</code>
    <dd>
-    <p>Push a copy of <var>object</var>'s contents to <var>stream</var>.
+    <p><span title=concept-enqueue-ReadableStream>Enqueue</span> a <code>Uint8Array</code> object
```

Comment:
Yes, I would say we want more flexibility to stream a file backed blob into memory.  Consider if someone is trying to stream a large zip off disk and pick out a single compressed item.  Requiring the entire zip to be in memory defeats much of the purpose of streams.